tox.ini

```diff
@@ -4,7 +4,7 @@
 # based on tests/docker/Dockerfile
 
 [tox]
-envlist = {unit,integration,functional}-py{37,38}-django{111,22}, javascript, docs
+envlist = {unit,integration,functional}-py{37,38}-django22, javascript, docs
```

Comment:
When do we start testing on Django 3.1? It's too late when 3.2 is out :)

Reply:
I suggest branching out from master right now, for this very purpose.